contrib/msoffice/src/com/sun/jna/platform/win32/COM/util/office/MSOfficeWordDemo.java

```diff
@@ -101,9 +101,6 @@ public void testMSWord() {
 			if (msWord != null) {
 				msWord.Quit();
 			}
-			if (null != factory) {
```

Comment:
While this is correct with regard to fixing the compile error, the example lacks proper COM initialization (the automatic init was removed together with the threading). (a Ole32.INSTANCE.CoInitializeEx/Ole32.INSTANCE.CoUninitialize() pair is missing)

At this point factory.disposeAll should be called to dispose all COM objects (I'm not sure how COM reacts to programm shutdown, but in any case a clean shutdown is preferred).